components/script/dom/bindings/trace.rs

```diff
@@ -586,6 +589,7 @@ unsafe_no_jsmanaged_fields!(MediaSessionActionType);
 unsafe_no_jsmanaged_fields!(MediaMetadata);
 unsafe_no_jsmanaged_fields!(WebrenderIpcSender);
 unsafe_no_jsmanaged_fields!(StreamConsumer);
+unsafe_no_jsmanaged_fields!(Animations);
```

Comment:
We should derive JSTraceable on the Animations type instead.

Reply:
Oh yeah, that's much cleaner. Fixed.